src/components/KYCWall/kycWallPropTypes.js

```diff
@@ -33,6 +44,9 @@ const defaultProps = {
     shouldListenForResize: false,
     isDisabled: false,
     chatReportID: '',
+    bankAccountList: {},
+    cardList: {},
```

Comment:
The default value should be `[]` based on the `PaymentUtils.hasExpensifyPaymentMethod` that use both props as the arguments.

Reply:
@Prince-Mendiratta To think more, it is the same result from `hasExpensifyPaymentMethod` because using `_.some`, and I just checked that on the Onyx, the default value for `bankAccountList` and `cardList` is an empty object 😅

Reply:
Let's have that empty object for similar with other usage and the default value from Onyx.